.github/workflows/build-package.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Package
+
+env:
+  MAIN_BRANCH: "all-azure_gdpr"
+  PACKAGE_CLOUD_REPO_NAME: "citusdata/azure"
+  PACKAGE_CLOUD_API_TOKEN: ${{ secrets.PACKAGE_CLOUD_API_TOKEN }}
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+  PACKAGING_SECRET_KEY: ${{ secrets.PACKAGING_SECRET_KEY }}
+  PACKAGING_PASSPHRASE: ${{ secrets.PACKAGING_PASSPHRASE }}
+on:
+  push:
+    branches:
+      - "**"
+
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    name: Build package
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
Why do we not fail fast? Any particular reason?

Reply:
Yes matrix jobs are independent from each other. So If a package job fails for an os version, I want other jobs continue. For some cases, there may be some os specific problems and I don't want these spesific problems to block other os jobs